assignment/index.html

```diff
@@ -19,7 +19,7 @@
     <!--Your code starts here-->
     <script>
       var map = L.map('map', {
-        center: [39.9522, -75.1639],
+        center: [39.9522, -72.0000],
```

Comment:
This change is confusing and counter productive.

It's good to play around with parameters to see what they do. 

But when I first opened up your map, it was in the middle of the ocean, and it looked like the map was not loading properly.

It'd be helpful if that map is centered on the appropriate zoom to level to the data you are tying to show.